ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/textinput/v1/textinput/textinput.html

```diff
@@ -27,27 +27,32 @@
      class="cmp-adaptiveform-textinput"
      data-cmp-is="adaptiveFormTextInput"
      id="${component.id}"
-     data-cmp-adaptiveformcontainer-path="${formstructparser.formContainerPath}">
+     data-cmp-adaptiveformcontainer-path="${formstructparser.formContainerPath}"
+     data-sly-test.longDescriptionId="${'{0}-{1}' @ format=[component.id, 'longDescription']}">
     <div data-sly-call="${label.label @componentId=component.id, labelValue=text.label.value, labelVisible=text.label.visible, bemBlock='cmp-adaptiveform-textinput'}" data-sly-unwrap></div>
-    <div data-sly-call="${questionMark.questionMark @componentId=component.id, longDescription=text.description, bemBlock='cmp-adaptiveform-textinput'}" data-sly-unwrap></div>
+    <div data-sly-call="${questionMark.questionMark @componentId=component.id, longDescription=text.description, label=text.label.value, bemBlock='cmp-adaptiveform-textinput'}" data-sly-unwrap></div>
     <textarea data-sly-test.useTextarea="${text.multiLine}"
-              aria-label="${text.label.visible && text.label.value}"
+              aria-label="${text.htmlScreenReaderText && text.screenReaderText != '$description' ? text.htmlScreenReaderText : text.label.value}"
+              aria-labelledby="${text.screenReaderText == '$description' ? longDescriptionId : ''}"
               class="cmp-adaptiveform-textinput__widget"
               name="${text.name}"
               readonly="${text.readOnly}"
               required="${text.required}"
               placeholder="${text.placeHolder}"
-              data-cmp-required-message="${text.constraintMessages.required}">${text.default}</textarea>
+              data-cmp-required-message="${text.constraintMessages.required}"
```

Comment:
pls remove **data-cmp-required-message**

Reply:
Also removed in PR https://github.com/adobe/aem-core-forms-components/pull/140 . Please review and merge https://github.com/adobe/aem-core-forms-components/pull/140 first and I will resolve the conflicts and reopen this PR.

Reply:
#193 was merged instead of #140